frontend/fe-evident/src/app/components/calendar/add-lecture/add-lecture-dialog/add-lecture-dialog.component.ts

```diff
@@ -66,15 +66,15 @@ export class AddLectureDialogComponent implements OnInit {
       lectureStartDate.setHours(lectureStartDate.getHours());
       const lectureStartTime = formValues.lectureStart;
       const [startHours, startMinutes] = lectureStartTime.split(':').map(Number); // Razdvajamo sate i minute i pretvaramo ih u brojeve
-      lectureStartDate.setHours(startHours);
+      lectureStartDate.setHours(startHours + 2);
```

Comment:
Jel ovo sigurno dobro? Hoće to u bazu onda spremit UTC vrijeme, ili ipak +00:02?